src/linear_algebra/bilinear_form.lean

```diff
@@ -779,6 +893,75 @@ end to_matrix
 
 end matrix
 
+namespace finsupp
+
+variables {α β : Type*}
+variables {M' : Type*} [add_comm_monoid M']
+variables {N : Type*} [has_zero N]
+
+lemma bilin_add_left' (A : α → β → N → (M' →+ R)) (p q : α →₀ M') (r : β →₀ N) :
```

Comment:
The name of this lemma seems a bit weird, since there's no mention of linearity at all in the statement or proof. Since this is just an auxiliary lemma to build `bilin_form_of`, can you mark it `private`, so that the name doesn't matter?

Reply:
I agree: I would go with either:
```suggestion
private lemma bilin_add_left' (A : α → β → N → (M' →+ R)) (p q : α →₀ M') (r : β →₀ N) :
```
or
```suggestion
lemma bilin_form_of_add_left_aux (A : α → β → N → (M' →+ R)) (p q : α →₀ M') (r : β →₀ N) :
```